### ISSUE TITLE
Unable to use `stringFromParts` with `string` generator parts

### ISSUE THREAD

**Issue by msievers** [feliixx/mgodatagen#92]
Hi!

First, thank you for this awesome tool which I use for many years now when it comes to MongoDB performance evaluations.

I'm currently trying to generate a field, which consists of a couple of random strings. My approach was to use `stringFromParts` together with `string` generators as parts. But unfortunately, I cannot get it to work. Here is a sample configuration file:

```json
[
  {
    "database": "someDatabase",
    "collection": "someCollection",
    "count": 10,
    "content": {
      "someString": {
        "type": "string",
        "minLength": 3,
        "maxLength": 5
      },
      "someStringFromParts": {
        "type": "stringFromParts",
        "parts": [
          {
            "type": "string",
            "minLength": 3,
            "maxLength": 5
          },
          {
            "type": "string",
            "minLength": 3,
            "maxLength": 5
          }
        ]
      }
    }
  }
]
```

which gives the following in the database

![Screenshot from 2022-09-01 11-46-38](https://user-images.githubusercontent.com/156371/187885830-eb34ed06-518f-46cd-b45f-20ea2e061dc0.png)

One can see, that the property `someString` is filled, `someStringFromParts` is not, although the used `string` generator settings are exactly the same.

Here is a playground with configuration above:

https://mongoplayground.net/p/1QLji3sXcT6

**Comment by feliixx** [comment]
Hi @msievers, 

Thanks for the report! You're right, it was a bug. It should be fixed by version [0.11.0](https://github.com/feliixx/mgodatagen/releases/tag/v0.11.0)

**Comment by msievers** [comment]
@feliixx Thank you very much for the quick reaction. This will instantly help me on Monday. Good work, great tool 🚀
